### PR TITLE
Extend checkDatabaseAllowUnknownTables to allow unknown composite types

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# hpqtypes-extras-1.8.0.0 (2019-05-21)
```
Comment (23Skidoo) [inline CHANGELOG.md]: Should be 1.9.0.0.

File: test/Main.hs [hunk 2]
```diff
@@ -749,35 +749,35 @@ migrationTest2 connSource =
     checkDatabase extrasOptions [] [] currentSchema
   assertNoException "checkDatabaseAllowUnknownTables runs fine \
                     \for consistent DB" $
-    checkDatabaseAllowUnknownTables extrasOptions [] [] currentSchema
+    checkDatabaseAllowUnknownObjects extrasOptions [] [] currentSchema
```
Comment (23Skidoo) [inline test/Main.hs]: Shouldn't we also add a test to this file that checks that unknown composites are accepted?

**Comment by arybczak** [comment]
I updated changelog and tests, should be good to merge now.

**Comment by 23Skidoo** [comment]
Travis is red, looks like it's due to a missing upper bound and a Haddock error.

**Comment by arybczak** [comment]
Ok, it's good now (travis failed for some but it's its fault).

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Pull request merged**